src/DataProtection/DataProtection.Autorest/custom/Cmdlets/Platform/BackupInstance/Stop-AzDataProtectionBackupInstanceProtection.ps1

```diff
@@ -104,10 +108,21 @@ function Stop-AzDataProtectionBackupInstanceProtection
             $null = $PSBoundParameters.Add("Parameter", $Parameter)
         }
 
-        if($PSBoundParameters.ContainsKey("Token"))
-        {            
-            $null = $PSBoundParameters.Remove("Token")
-            $null = $PSBoundParameters.Add("Token", "Bearer $Token")
+        $hasToken = $PSBoundParameters.Remove("Token")
+        $hasSecureToken = $PSBoundParameters.Remove("SecureToken")
+        if($hasToken -or $hasSecureToken)
+        {   
+            if($hasSecureToken -and $hasToken){
+                throw "Both Token and SecureToken parameters cannot be provided together"
+            }
+            elseif($hasToken){
+                Write-Warning -Message 'The Token parameter is deprecated and will be removed in future versions. Please use SecureToken instead.'
+                $null = $PSBoundParameters.Add("Token", "Bearer $Token")
+            }
+            else{
+                $plainToken = ConvertFrom-SecureString $secureToken -AsPlainText
```

Comment:
`-AsPlainText` is not supported on PowerShell 5.1. @dolauli please help suggest an alterative

Reply:
Please refer to https://github.com/Azure/azure-powershell/blob/generation/src/ContainerRegistry/ContainerRegistry.Autorest/utils/Unprotect-SecureString.ps1 for instructions on converting a secure string to a regular string, compatible with both Windows PowerShell and PowerShell 7.

Reply:
made changes to accommodate this